harness/determined/cli/master.py

```diff
@@ -11,54 +11,62 @@
 
 @authentication.required
 def config(args: Namespace) -> None:
-    if args.log:
-        if args.level and args.color:
-            if args.color.lower() == "on":
-                master_config = bindings.v1Config(
-                    log=bindings.v1LogConfig(level=args.level, color=True)
-                )
-            elif args.color.lower() == "off":
-                master_config = bindings.v1Config(
-                    log=bindings.v1LogConfig(level=args.level, color=False)
-                )
-            else:
-                raise CliError(
-                    "Error: Invalid value for --color provided: "
-                    + args.color
-                    + ". This optional parameter only accepts 'on' and 'off' values."
+    resp = bindings.get_GetMasterConfig(cli.setup_session(args)).config
+    if args.json:
+        determined.cli.render.print_json(resp)
```

Comment:
why not import the module instead of using the full path?

Reply:
done